src/main/java/teammates/ui/controller/InstructorFeedbackAbstractAction.java

```diff
@@ -0,0 +1,168 @@
+package teammates.ui.controller;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.HashMap;
+import java.util.List;
+
+import teammates.common.datatransfer.CourseAttributes;
+import teammates.common.datatransfer.FeedbackSessionAttributes;
+import teammates.common.datatransfer.FeedbackSessionType;
+import teammates.common.datatransfer.InstructorAttributes;
+import teammates.common.util.Const;
+import teammates.common.util.EmailType;
+import teammates.common.util.Sanitizer;
+import teammates.common.util.TimeHelper;
+
+public abstract class InstructorFeedbackAbstractAction extends Action {
+
+    /**
+     * Common method to Get the feedback data
+     */
+    protected FeedbackSessionAttributes extractFeedbackSessionDataHelper(FeedbackSessionAttributes newSession) {
+
+        return this.extractFeedbackSessionDataHelper(newSession);
+    }
+
+    protected FeedbackSessionAttributes extractFeedbackSessionDataHelper(
+            FeedbackSessionAttributes newSession, List<String> sendRemainderEmailsList) {
+        return this.extractFeedbackSessionDataHelper(newSession, sendRemainderEmailsList);
```

Comment:
@wkurniawan07 I believe followings are the reasons for having 2 methods : 

* It affects the code readability. Suppose if someone in the future wants to some changes then    dev won't understand why this(`List<String>` and `String[]` )are at the start because the `const` used in creating array and the array name `senderReminderEmail` which says more than enough.

  
* First i wasn't sure whether the order of attributes creation matters but now due to the failing tests and the build log it matters.  

* Since the method is `extractFeedbackSessionData` why do we have to send the reminder email about the feedback we need to send it to only those who haven't submitted the feedback. If we create the `List<String>` and `String[]` at starting this happens.

* If you wonder why can't we add it at the end there are some attributes like `setCreatedTime` which has to be created in the start.

So i think we can't handle all these and have a single method.